flake8_use_fstring/prefix.py

```diff
@@ -0,0 +1,80 @@
+import re
+import token as _token
+from typing import Iterator, Tuple
+
+from flake8.options.manager import (
+    OptionManager as _OptionManager,
+)
+
+from .base import (
+    BaseLogicalLineChecker as _Base,
+)
+
+FSTRING_REGEX = re.compile(r'^([a-zA-Z]*?[fF][a-zA-Z]*?){1}["\']')
+NON_FSTRING_REGEX = re.compile(
+    r'^[a-zA-Z]*(?:\'\'\'|\'|"""|")(.*?{.+?}.*)(?:\'|\'\'\'|"|""")$')
+
+
+class MissingPrefixDetector(_Base):
+    name = 'use-fstring-prefix'
+    version = '1.0'
+    greedy = '0'
+    enabled = False
+
+    def __getitem__(self, i: int) -> bool:
+        if (not self.enabled):
+            return False
+
+        token = self.tokens[i]
+        if token.exact_type != _token.STRING:
+            return False
+
+        if FSTRING_REGEX.search(token.string):  # already is an f-string
+            return False
+
+        # look ahead for % or .format and skip if present
```

Comment:
We probably can use an option to toggle this behavior? But, I need to think this through and cover all different use cases.

Reply:
I still don't see the value of ignoring that check, but I went ahead and added the option to disable it. I left it on by default.